index.bs

```diff
@@ -127,31 +127,29 @@ systems.
 A <dfn export id=file>file entry</dfn> additionally consists of
 <dfn for="file entry" export>binary data</dfn> (a [=byte sequence=]), a
 <dfn for="file entry">modification timestamp</dfn> (a number representing the number of milliseconds since the <a spec=FileAPI>Unix Epoch</a>),
-a <dfn for="file entry">lock</dfn> (a string that may exclusively be "`open`", "`taken-exclusive`" or "`taken-shared`")
-and a <dfn for="file entry">shared lock count</dfn> (a number representing the number shared locks that are taken at a given point in time).
+a <dfn for="file entry">lock</dfn> (a [=string=] that may exclusively be "`open`", "`exclusive`", "`writable-siloed`",
+"`sync-access-handle-read-only`", "`sync-access-handle-read-write-unsafe`")
```

Comment:
Thoughts on defining a "file system lock type" before this, which "file entry/lock" can then refer to? Then also the "take a lock" algorithm can take a "file system lock type" and just assert its not passed "open"